packages/classes/README.md

```diff
@@ -21,3 +21,4 @@ npm install --save-dev @tvkitchen/base-classes
 ## About the Classes
 
 * [Payload](docs/PAYLOAD.md): used to pass data between components.
+* [PayloadArray](docs/PAYLOAD_ARRAY.md): used to store and filter sets of payloads.
```

Comment:
Minor point: [`Set`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) is an actual iterable type in JavaScript. We may want to avoid using it as a generic name. Perhaps "filter collections of payloads" would be better here? (Or "series of", which you use in the JSDoc?)

Reply:
BUT IT IS LOWER CASE.  (sounds good)